zc_common/jwt_auth/authentication.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, **kwargs):
             setattr(self, kwarg, kwargs[kwarg])
 
     def is_authenticated(self):
-        return 'user' in self.roles or 'service' in self.roles
+        return True
```

Comment:
@birkholz I changed the line because that is in fact what permissions are doing (checking roles). Also, I am returning `True` because all requests must be authenticated by 'JWTAuthentication' class. Finally, I thought `guest` was an attribute in `mp-users` not a role?